src/main/java/com/duke/bot/Storage.java

```diff
@@ -0,0 +1,45 @@
+package com.duke.bot;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Represents the permanent storage of the list of tasks managed by Duke Bot into the hard drive.
+ */
+public class Storage {
+    private static final String DESTINATION_PATH = "C:\\tasks.txt";
+    private Storage() {
+    }
+
+    /**
+     * Creates a Storage File
+     */
```

Comment:
The Javadoc should provide @return to indicate the return statement of the method.